jwala-services/src/main/java/com/cerner/jwala/service/initializer/JGroupsClusterInitializer.java

```diff
@@ -61,6 +57,13 @@ public void afterPropertiesSet() throws Exception {
                 }
                 coordinatorIP = getIPAddressFromHostname(jgroupsCoordinatorHostname);
             }
+
+            System.setProperty(JAVA_NET_PREFER_IPV4_STACK, jgroupsJavaNetPreferIPv4Stack);
+            System.setProperty(JGROUPS_BIND_ADDR, coordinatorIP.getIpAddress().getHostAddress());
```

Comment:
Can you add null check for getIpAddress to avoid NPE? I am not sure if this could be null.